src/components/Inputs/textField/TextField.scss

```diff
@@ -20,6 +19,15 @@
     }
 
     &:focus {
-        border: 1px solid $primary-main;
+        border: 2px solid $primary-main;
```

Comment:
to nebo tak preprosto šlo. Primerjaj tvojo in njihovo animacijo. Pri tvoji se velikost inputa spremeni, pri mui.com text field se velikost inputa ne spremeni, čeprav se border poveča.

Lahko rešiš tak kot oni ali pa naredi na 1px